kyuubi-common/src/main/scala/org/apache/kyuubi/service/BackendService.scala

```diff
@@ -91,7 +91,8 @@ trait BackendService {
       foreignTable: String): OperationHandle
   def getQueryId(operationHandle: OperationHandle): String
 
-  def getOperationStatus(operationHandle: OperationHandle): OperationStatus
+  def getOperationStatus(operationHandle: OperationHandle, maxWait: Long): OperationStatus
```

Comment:
after second thought, how about `maxWait: Option[Long] = None` ? then we do not need a new method `getOperationStatusTimeout`